av.c

```diff
@@ -95,9 +95,10 @@ Perl_av_extend(pTHX_ AV *av, SSize_t key)
 }    
 
```

Comment:
note to reviewer, as stated in the discussion you have to commits one for `s{\t}{\s}` and one with the core changes.
Better reviewing the second commit by itself